core/cas-server-core-events-api/src/main/java/org/apereo/cas/support/events/listener/CasAuthenticationAuthenticationEventListener.java

```diff
@@ -40,6 +47,9 @@ public class CasAuthenticationAuthenticationEventListener implements CasAuthenti
 
     private final GeoLocationService geoLocationService;
 
+    @Setter
+    private LogoutManager logoutManager;
```

Comment:
This cannot be; I suspect you need this for tests somehow, but the final must be final without a setter.

Reply:
It's a mistake, it is not necessary. I will revert it.